packages/react/src/ReactNodeViewRenderer.tsx

```diff
@@ -76,7 +78,8 @@ export class ReactNodeView<
       getPos: () => this.getPos(),
       updateAttributes: (attributes = {}) => this.updateAttributes(attributes),
       deleteNode: () => this.deleteNode(),
-    } satisfies NodeViewProps
+      ref: React.createRef<HTMLElement>(),
```

Comment:
Consider importing entities from the React module instead of importing the whole React namespace. `import { createRef } from 'react'` insead of `import React from 'react'`